packages/orchestrator/lib/clients/client.ts

```diff
@@ -79,6 +85,50 @@ export class OrchestratorClient {
         }
     }
 
+    // TODO
+    public async cancelSync(props: CancelSyncProps): Promise<VoidReturn> {
+        return Err({
+            name: 'not_implemented',
+            message: 'Not implemented',
+            payload: JSON.stringify({ scheduleName: props.scheduleName })
+        });
+    }
+
+    // TODO
+    public async pauseSync(props: PauseSyncProps): Promise<VoidReturn> {
+        return Err({
+            name: 'not_implemented',
+            message: 'Not implemented',
+            payload: JSON.stringify({ scheduleName: props.scheduleName })
+        });
+    }
+
+    // TODO
+    public async unpauseSync(props: UnpauseSyncProps): Promise<VoidReturn> {
+        return Err({
+            name: 'not_implemented',
+            message: 'Not implemented',
+            payload: JSON.stringify({ scheduleName: props.scheduleName })
+        });
+    }
+
+    public async executeSync(props: ExecuteSyncProps): Promise<VoidReturn> {
+        const res = await this.routeFetch(postRecurringRunRoute)({
+            body: {
+                scheduleName: props.scheduleName
+            }
+        });
+        if ('error' in res) {
+            return Err({
+                name: res.error.code,
+                message: res.error.message || `Error creating recurring schedule`,
+                payload: JSON.stringify(props)
```

Comment:
any reason to always stringify payload? feels like it's what give us the \\\\\\" in the recent errors we had